app/main/processes/process_manager.py

```diff
@@ -46,15 +46,15 @@ def create(
         # Create a new process
         fl_process = self._processes.register(name=name, version=version)
 
-        # Register client plans
-        plans.register(fl_process, client_plans, avg_plan=False)
-
         # Register client protocols
         protocols.register(fl_process, client_protocols)
 
         # Register Server avg plan
         plans.register(fl_process, server_avg_plan, avg_plan=True)
 
+        # Register client plans
+        plans.register(fl_process, client_plans, avg_plan=False)
+
```

Comment:
I have no idea why, but if client plans are registered first, fl_process_id of the first plan is not populated in the DB.
Moving it after server avg plan fixes the problem.